db/upgrade-X.X.X-X.Y.Z.sql

```diff
@@ -17,3 +17,74 @@ SET @SUBMINOR_VERSION = 9;
 
 SET @VERSION_INT = @MAJOR_VERSION << 16 | @MINOR_VERSION << 8 | @SUBMINOR_VERSION;
 
+--
+-- Table structure for table `node_option82`
+--
+
+CREATE TABLE `node_option82` (
+  `node_option82_id` int NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  `created_at` TIMESTAMP NOT NULL,
+  `mac` varchar(17) NOT NULL,
+  `option82_switch` varchar(17) NULL,
+  `switch_id` varchar(17) NULL,
+  `port` varchar(8) NOT NULL default '',
+  `vlan` varchar(255) default NULL,
+  `circuit_id_string` varchar(255) default NULL,
+  `module` varchar(255) default NULL,
+  `host` varchar(255) default NULL,
+  UNIQUE KEY mac (mac)
+) ENGINE=InnoDB;
+
+--
+-- Table structure for table `node_option82_history`
+--
+
+CREATE TABLE `node_option82_history` (
+  `node_option82_id` int NOT NULL PRIMARY KEY AUTO_INCREMENT,
```

Comment:
AUTO_INCREMENT ?
I think we should keep the "original" node_option82_id.
See next comment